Xamarin.Forms.Core/Slider.cs

```diff
@@ -49,6 +50,10 @@ public class Slider : View, IElementConfiguration<Slider>
 
 		public static readonly BindableProperty ThumbImageProperty = BindableProperty.Create(nameof(ThumbImage), typeof(FileImageSource), typeof(Slider), default(FileImageSource));
 
+		public static readonly BindableProperty DragStartedCommandProperty = BindableProperty.Create(nameof(DragStartedCommand), typeof(ICommand), typeof(Slider), default(ICommand));
+
+		public static readonly BindableProperty DragCompletedCommandProperty = BindableProperty.Create(nameof(DragCompletedCommand), typeof(ICommand), typeof(Slider), default(ICommand));
```

Comment:
I would not create commands for these events (we don't have one for ValueChanged)

Reply:
@StephaneDelcroix Can you explain further? This would deviate from how XF normally handles UI controls which have user interaction ie. Button, GestureRecognizer's etc.

We don't have a command for ValueChanged as this is simply in response to a property changing, however drag started/completed are user interactions similar to Button's clicked event and TapGestureRecognizer's Tapped event. Both of these have commands for MVVM support.

I guess you could use EventToCommandBehavior, but it would seem consistent to utilise commands for the UI drag events to allow simply binding to a view model.

Reply:
¯\_(ツ)_/¯